api/query_test.go

```diff
@@ -82,8 +82,9 @@ type testSliceAxis struct {
 }
 
 type testSliceMetadata struct {
-	Axis   []testSliceAxis `json:"axis"   binding:"required"`
-	Format int             `json:"format" binding:"required"`
+	X      testSliceAxis `json:"x"      binding:"required"`
+	Y      testSliceAxis `json:"y"      binding:"required"`
```

Comment:
I'm torn. I really like the shape entry, as it makes for nice assembly of the data on call site. E.g. `np.ndarray(meta['shape'], meta['format'], data)`
However, that duplicated data, which I'm not a huge fan of

Reply:
What will duplicate data?
I am thinking to have shape _instead of_ X and Y. The whole Axis information is in metadata anyway. Shape was the main point why we decided to return this thing twice, right?

Reply:
We want to keep the axis information. Otherwise you'd have to do 2 queries, and then manually find the axises that correspond to the slice. Which is easy to mess up. That's why it was introduced in the first place